spec/unit/lib/cloud_controller/diego/app_recipe_builder_spec.rb

```diff
@@ -216,7 +221,8 @@ module Diego
               global_environment_variables: env_vars,
               privileged?:                  false,
               ports:                        lrp_builder_ports,
-              port_environment_variables:   expected_action_environment_variables,
+              port_environment_variables:   expected_action_port_variables,
```

Comment:
Since we stub out the `Buildpack::DesiredLrpBuilder` with an instance double here, we should not have to touch the `app_recipe_builder_spec` since our changeset is isolated within the `Buildpack::DesiredLrpBuilder`. This statement only holds true if the two methods are collapsed as mentioned above.